escuela-nacional-de-antropologia-e-historia-author-date.csl

```diff
@@ -486,15 +486,13 @@
     </layout>
   </citation>
   <!-- Para generar la bibliografía -->
-  <bibliography et-al-min="4" et-al-use-first="3" subsequent-author-substitute="" entry-spacing="0">
+  <bibliography et-al-min="4" et-al-use-first="3" subsequent-author-substitute="" entry-spacing="0" line-spacing="1">
     <sort>
       <key macro="contributors"/>
       <key macro="date"/>
     </sort>
     <layout suffix=".">
-      <group display="block">
-        <text macro="contributors"/>
-      </group>
+      <text macro="contributors" suffix='&#xD;'/>
```

Comment:
@adam3smith, oh, it looks like this has already been discussed at https://github.com/citation-style-language/styles/pull/2199